src/clr-base/index.ts

```diff
@@ -4,4 +4,5 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-export * from './lib/utils/css-vars/css-vars';
+export * from './utils';
+export * from './components/clr-wc-element/clr-wc-element';
```

Comment:
Main entry point exports all entry points (utils + components). This will allow users to import just individual components or all components.

```typescript
// import single component
import '@clr/base/components/clr-wc-element';

// import all components
import '@clr/base';
```

Reply:
🏆

Reply:
QQ: don't we need to support the following? Because one WC could have multiple files that could be exported as a module?
```
export * from './components/clr-wc-element';
```